database/migrations/2023_06_20_123227_change_image_column_nullable_in_places_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ChangeImageColumnNullableInPlacesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('places', function (Blueprint $table) {
+            $table->string('image',255)->nullable()->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('places', function (Blueprint $table) {
+            DB::table('places')->whereNull('image')->update(['image' => '']);
```

Comment:
just move this outside the schema

Reply:
I moved the updating eloquent outside the schema